{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/production.py

```diff
@@ -13,6 +13,15 @@
 
 ALLOWED_HOSTS = ['*']
 
+# Enable offline compression of CSS/JS
```

Comment:
Make these the default values in `base.py`. Set them to development-friendly values in `local.py`.